spec/cell_spec.rb

```diff
@@ -36,20 +36,19 @@ def cell(options={})
     end
 
     it "should convert nil, Numeric, and Date values to strings" do
-      [nil, 123, 123.45, Date.today].each do |value|
+      [nil, 123, 123.45, Date.today, Time.new].each do |value|
         c = @pdf.cell(:content => value)
         expect(c).to be_a_kind_of Prawn::Table::Cell::Text
         expect(c.content).to eq value.to_s
       end
     end
 
-    it "should allow inline styling with a hash argument" do
-      # used for table([[{:text => "...", :font_style => :bold, ...}, ...]])
-      c = Prawn::Table::Cell.make(@pdf,
-                                  {:content => 'hello', :font_style => :bold})
-      expect(c).to be_a_kind_of Prawn::Table::Cell::Text
-      expect(c.content).to eq "hello"
-      expect(c.font.name).to eq 'Helvetica-Bold'
+    it "should convert nil, Numeric, and Date values to strings when value is extracted from options" do
+      [nil, 123, 123.45, Date.today, Time.new].each do |value|
+        c = Prawn::Table::Cell.make(@pdf, {}, { :content => value })
+        expect(c).to be_a_kind_of Prawn::Table::Cell::Text
+        expect(c.content).to eq value.to_s
+      end
```

Comment:
This looks a lot like the previous spec. Does it cover anything not covered by the other one?

Reply:
This spec tests an alternate path to the previous spec.  It's the whole point of #119.

The previous spec tests normalization of the passed in content argument, if it's not a Hash.  But there's an alternate path for getting the content value that was not being tested for normalization.  And the previous code wasn't normalizing it.

It content is a hash then the code:

1. Checks to see if this is supposed to be an image and, if so, returns that
2. Otherwise, merges the content hash into options
3. And then uses the :content value from that merged hash

In the existing code the value retrieved from (3) is not stringified when it should be.  Hence the error.  This spec is necessary to test this path.